### ISSUE TITLE
ERROR in mlno_get_root_eind_to_outgroup: Outgroup does not exist

### ISSUE THREAD

**Issue by RvV1979** [sriramlab/OrientAGraph#10]
I am running both TreeMix and OrientAGraph on a dataset that includes populations from two outgroup species. Using TreeMix, setting `-root out1,out2` works. However, using OrientAGraph, using `-root out1,out2` throws the following error: 
`ERROR in mlno_get_root_eind_to_outgroup: Outgroup does not exist!`
Note that using either -root out or -root out2 individually works. Results then also show the other outgroup population as the sister to the ingroup, so appropriate rooting based on both outgroups should be possible. 

I am guessing that `mlno_get_root_eind_to_outgroup` cannot root the tree with multiple outgroup populations. Is that correct, and is there a way around this? 

Thanks

**Comment by ekmolloy** [comment]
You are correct. I should be able to update the code to allow multiple outgroups (under the condition that TreeMix can root the starting tree at those outgroups). If you run OrientAGraph with -freq2stat, you will get upload the two matrices (you can rename the pop names to something ambiguous like A, B, C, D...). If you upload the matrices here or send them to at ekmolloy@umd.edu, I can try to get the code to work for your data with 2 outgroups in the debug branch.
